README.md

```diff
@@ -122,12 +151,11 @@ To run the project locally, use the following commands:
    ```
 
 #### Frontend
-1. Navigate to the `nextjs-frontend` directory.
-2. Install dependencies (if not already installed):
+1. Navigate to the `nextjs-frontend` directory and run:
    ```bash
-   npm install
+   pnpm install
    ```
-3. Start the Next.js development server:
+2. Start the Next.js development server:
```

Comment:
This part of running should be separate from install. Put this under the section `Running the Application`.
Also, move `Pre-Commit Setup` to the bottom after `Testing` secion.